docs/about/contributing.md

```diff
@@ -41,148 +41,158 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
+* **[meta-cli][meta-cli-repo]**: A Go-based CLI for reading/writing information from the metadata
 
 ### Executors
 
 An executor is used to manage build containers for any given job. Several implementations of executors have been created. All are designed to follow a common interface. Executor implementations are written in node:
 
-* **[executor-base][executor-base-repo]**: Common interface [![Version][executor-base-npm-image]][executor-base-npm-url]
-* **[executor-docker][executor-docker-repo]**: Docker implementation [![Version][executor-docker-npm-image]][executor-docker-npm-url]
-* **[executor-j5s](https://github.com/screwdriver-cd/executor-j5s)**: Jenkins implementation [![Version][executor-j5s-npm-image]][executor-j5s-npm-url]
-* **[executor-k8s][executor-k8s-repo]**: Kubernetes implementation [![Version][executor-k8s-npm-image]][executor-k8s-npm-url]
+* **[executor-base][executor-base-repo]**: Common interface
+* **[executor-docker][executor-docker-repo]**: Docker implementation
+* **[executor-j5s](https://github.com/screwdriver-cd/executor-j5s)**: Jenkins implementation
+* **[executor-k8s][executor-k8s-repo]**: Kubernetes implementation
+* **[executor-k8s-vm][executor-k8s-vm-repo]**: Kubernetes VM implementation
+
+The [executor router][executor-router-repo] is a generic executor plugin that routes builds to a specified executor.
 
 ### Models
 
-The object models provide the definition of the data that is stored in data stores. This is done in two parts:
+The object models provide the definition of the data that is saved in datastores (analogous to databases). This is done in two parts:
 
-* **[data-schema][dataschema-repo]**: Schema definition with *[Joi](https://www.npmjs.com/package/joi)* [![Version][dataschema-npm-image]][dataschema-npm-url]
-* **[models][models-repo]**: Specific business logic around the data schema [![Version][models-npm-image]][models-npm-url]
+* **[data-schema][dataschema-repo]**: Schema definition with *[Joi](https://www.npmjs.com/package/joi)*
+* **[models][models-repo]**: Specific business logic around the data schema
 
 ### Datastores
 
 A datastore implementation is used as the interface between the API and a data storage mechanism. There are several implementations written in node around a common interface.
 
-* **[datastore-base][datastore-base-repo]**: Common interface [![Version][datastore-base-npm-image]][datastore-base-npm-url]
-* **[datastore-sequelize][datastore-sequelize-repo]**: Mysql, postgres, sqlite3 and mssql implementation [![Version][datastore-sequelize-npm-image]][datastore-sequelize-npm-url]
+* **[datastore-base][datastore-base-repo]**: Base class defining the interface for datastore implementations
+* **[datastore-sequelize][datastore-sequelize-repo]**: MySQL, PostgreSQL, SQLite3, and MS SQL implementations
+* **[datastore-dynamodb][datastore-dynamodb-repo]**: DynamoDB implementation, with [dynamic-dynamodb][dynamic-dynamodb-repo] being used to create the datastore tables for it
```

Comment:
When speaking with @FenrirUnbound , we decided to keep the repo in case anyone else wanted to work on it (i.e. just because we don't use it anymore, doesn't mean we should get rid of it).